k8s/superset/superset-deployment.yaml

```
@@ -28,8 +28,9 @@ spec:
           ports:
             - containerPort: 8088
           envFrom:
-            - configMapRef:
-                name: superset-config
+          - configMapRef:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)